modules/dataquery/jsx/react.app.js

```diff
@@ -103,7 +103,7 @@ var DataQueryApp = React.createClass({
         var promises = [];
         var that = this;
         for(var key in this.state.queryIDs){
-            console.log(this.state.queryIDs[key][0]);
+            //console.log(this.state.queryIDs[key][0]);
```

Comment:
You are keeping the console log because this is just a quick fix and will be removed when the module is rewritten, right?

Reply:
@intralizee or could we remove this comment? if not, is there a reason we want to keep it?